File: MuayeneYonetimPortali/MuayeneYonetimPortali.Web/Modules/ServerTypes/Tanimlamalar/IsActive.ts
import { Decorators } from "@serenity-is/corelib";

export enum IsActive {
    Pasif = 0,
    Aktif = 1,
    Silinmis = 2
}
Decorators.registerEnumType(IsActive, 'MuayeneYonetimPortali.Tanimlamalar.IsActive', 'Tanimlamalar.IsActive');

File: MuayeneYonetimPortali/MuayeneYonetimPortali.Web/Modules/Tanimlamalar/Enums/IsActive.cs
namespace MuayeneYonetimPortali.Tanimlamalar;

[EnumKey("Tanimlamalar.Enums.IsActive")]
public enum IsActive
{
    [Description("Pasif")]
    Pasif = 0,
    [Description("Aktif")]
    Aktif = 1,
    [Description("Silinmiş")]
    Silinmis = -1
}